src/RealtimeClient.ts

```diff
@@ -119,6 +124,8 @@ export default class RealtimeClient {
    * @param options.encode The function to encode outgoing messages. Defaults to JSON: (payload, callback) => callback(JSON.stringify(payload))
    * @param options.decode The function to decode incoming messages. Defaults to Serializer's decode.
    * @param options.reconnectAfterMs he optional function that returns the millsec reconnect interval. Defaults to stepped backoff off.
+   * @param options.worker Use Web Worker to set a side flow. Defaults to false.
+   * @param options.workerUrl The URL of the worker script. Defaults to https://realtime.supabase.com/worker.js that includes a heartbeat event call to keep the connection alive.
```

Comment:
We need to proxy this through `project_ref.supabase.co/realtime/v1/worker.js` before this gets into main.

Reply:
depending on how we pass it in just be careful when folks are developing locally and their project ref is like `dev_tenant` or something. might mess things up on our end.